### PR TITLE
Improve model variable naming - CLIP [TF] 

### PR DESCRIPTION
**Pull request by bhavika**
# What does this PR do?

Uses the @unpack_inputs decorator instead of `input_processing` for CLIP (TF). See https://github.com/huggingface/transformers/issues/16051

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Tagging @Rocketknight1 for review! 👋🏽

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
cc @patil-suraj

**Review by gante — APPROVED** [review]
Thank you so much for your contribution <3 Can you please:
1. confirm that the tests pass (`RUN_SLOW=1 py.test -vv tests/[model_name]/test_modeling_tf_[model_name].py`)
2. address my other comment

(Will merge as soon as I get a yes for the two points above)

**Comment by gante** [comment]
Also, making a cheeky edit on your PR description -- I'm swapping `Fixes` with `See` (`Fixes` closes the associated issue)

File: src/transformers/models/clip/modeling_tf_clip.py [hunk 1]
```diff
@@ -1109,6 +1048,7 @@ def __init__(self, config: CLIPTextConfig, *inputs, **kwargs):
 
     @add_start_docstrings_to_model_forward(CLIP_TEXT_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @replace_return_docstrings(output_type=TFBaseModelOutputWithPooling, config_class=CLIPTextConfig)
+    @unpack_inputs
```
[the remark this answers is not in the document]
Comment (bhavika) [reply src/transformers/models/clip/modeling_tf_clip.py]: Fixed in https://github.com/huggingface/transformers/pull/16128/commits/2840ef018073dd7df88a2e85e193dfbca775b88b

**Comment by bhavika** [comment]
Thanks for the review @gante! Made changes in https://github.com/huggingface/transformers/pull/16128/commits/2840ef018073dd7df88a2e85e193dfbca775b88b and I'm confirming that `RUN_SLOW=1 py.test -vv tests/[model_name]/test_modeling_tf_[model_name].py` works for `clip`.

**Pull request merged**